README.md

```diff
@@ -1,22 +1,111 @@
 # Rokt iOS SDK
 
-The Rokt iOS SDK enables you to integrate Rokt into your native iOS mobile apps to drive more value from—and for—your customers. The SDK is built to be lightweight, secure, and simple to integrate and maintain, resulting in minimal lift for your engineering team.
+The Rokt iOS SDK enables you to integrate Rokt into your native iOS mobile apps to drive more value from, and for, your customers. The SDK is built to be lightweight, secure, and simple to integrate and maintain, resulting in minimal lift for your engineering team.
 
-## Swift Package Manager Integration
+## Resident Experts
 
-The [Swift Package Manager](https://swift.org/package-manager/) is a tool for automating the distribution of Swift code and is integrated into the `swift` compiler.
+- Emmanuel Tugado - emmanuel.tugado@rokt.com
```

Comment:
I would suggest removing our name as the readme is going to be shipped with the SDK and we don't want to have individual names there.